### PR TITLE
[Crane] Extract String resources

### LINKED ISSUE #612: [Crane] Use String resources
Many strings in the app are hard-coded. They should use the Resources instead.

### PR DESCRIPTION
**Pull request by riggaroo**
## Description ✏️

Fixes #612 - extract all strings to resources instead of them being hardcoded.

### CODE REVIEW

File: Crane/app/src/main/java/androidx/compose/samples/crane/home/SearchUserInput.kt [hunk 1]
```diff
@@ -72,8 +74,10 @@ fun PeopleUserInput(
         val tint = tintPeopleUserInput(transitionState)
 
         val people = peopleState.people
+        val resources = LocalContext.current.resources
+        val peopleString = resources.getQuantityString(R.plurals.number_adults_selected, people, people, titleSuffix)
```
Comment (bentrengrove) [inline Crane/app/src/main/java/androidx/compose/samples/crane/home/SearchUserInput.kt:78]: Plural support was added in the latest version of Compose, good place to try it!

**Comment by bentrengrove** [comment]
Looks like it needs a spotless run.
`./gradlew :app:spotlessApply`

File: Crane/app/src/main/java/androidx/compose/samples/crane/home/SearchUserInput.kt (hunk 1 above)
Comment (riggaroo) [reply Crane/app/src/main/java/androidx/compose/samples/crane/home/SearchUserInput.kt:78]: Cool @bentrengrove, think it'll be a good place for me to also to try update the docs then? https://developer.android.com/jetpack/compose/resources#string-plurals

Comment (bentrengrove) [reply Crane/app/src/main/java/androidx/compose/samples/crane/home/SearchUserInput.kt:78]: Definitely!

Comment (riggaroo) [reply Crane/app/src/main/java/androidx/compose/samples/crane/home/SearchUserInput.kt]: Updated this PR with the change. [d5c2732](https://github.com/android/compose-samples/pull/765/commits/d5c2732bec52ef0c0a91969eed8b0aa80119ae8f)

**Review by bentrengrove — APPROVED** [review]
Nice!

**Pull request merged**